images/vct/Dockerfile

```diff
@@ -29,12 +22,13 @@ ARG TARGETARCH
 RUN echo ${TARGETOS}
 RUN echo ${TARGETARCH}
 
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o build/bin/vct cmd/vct/main.go
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o /usr/bin/vct cmd/vct/main.go
 
-ARG ALPINE_VER
+FROM scratch
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
I would personally use a second multi-stage pure alpine image to copy these files from.

Reply:
ok, I will do that in the follow up PR